terraform/envs/dev/terraform.tfvars

```diff
@@ -32,4 +32,7 @@ show_ui_elements = ""
 
 # Storage
 bucket = "amplication-artifacts"
-default_disk = "gcs"
\ No newline at end of file
+default_disk = "gcs"
+
+# Apps Cloud Build
+apps_gcp_project_id = "amplication"
```

Comment:
I think the data on the deployment environment should be taken from a list in the DB. initially it will be a table with a single record (different between prod, staging, and dev) - but in the future we will be able to have multiple projects and select the right one dynamically.

Reply:
Duplicate, answered above.